lib/utils.ts

```diff
@@ -53,98 +46,17 @@ export function generateUUID(): string {
   });
 }
 
-function addToolMessageToChat({
```

Comment:
FYI - with the recent update on the `main` branch, the `addToolMessageToChat` and `sanitizeResponseMessages` functions are no longer used.

Hence, I've removed it, but @jeremyphilemon let me know if you plan on using this for something else in the future and I'll add it back in if needed.

Thanks